ssv/validator_registration.go

```diff
@@ -60,6 +61,20 @@ func (r *ValidatorRegistrationRunner) ProcessPreConsensus(signedMsg *types.Signe
 		return nil
 	}
 
+	// only 1 root, verified in basePreConsensusMsgProcessing
```

Comment:
why?
Was there an explanation in the SIP?

Reply:
Because `PartialSignatureMessages` contains only one message for this runner according to https://github.com/nkryuchkov/ssv-spec/blob/mev-support-2/ssv/validator_registration.go#L116. This particular check is copied from the aggregator's and proposer's `ProcessPreConsensus`

Reply:
AFAIK there is no explanation in the SIP but I'm not sure if we need it because I think it's rather implementation-specific. 

@moshe-blox As you wrote the SIP, what's your opinion on this?

Reply:
@GalRogozinski @nkryuchkov Gal spoke with me, I think we only take 1 root because we've signed only 1 message, is that right?